### ISSUE TITLE
Lint  <head><title> tag looking for curlies

### ISSUE THREAD

**Issue by kiwiupover** [ember-template-lint/ember-template-lint#654]
The `<title>` tag will print any html characters is side the tag.  When using fastboot and rehydration the `<title>` tag with dynamic attributes glimmer will add rehydration tags `<!--%+b:9%-->` inside the <title>. 

https://cl.ly/1r3h03190G3x

The lint rule should alert the dev that there could be an issue. 

https://dev.w3.org/html5/spec-preview/tokenization.html#rcdata-end-tag-name-state

**Comment by CvX** [comment]
I've had this issue, but I'd rather have it fixed in Glimmer VM, possibly by rendering rehydration tags only outside `<title>` (and `<script>`!) than have a lint rule (which would alert a false-positive when rehydration rendering isn't enabled).
Or do you think of this rule as a kind of a stop-gap?

**Comment by kiwiupover** [comment]
I agree with you @CvX a fix in glimmer would be better for sure. This could be a temporary fix until the upstream versions are fixed.

https://github.com/glimmerjs/glimmer-vm/issues/796

**Comment by Turbo87** [comment]
@kiwiupover I don't understand this. Can you give an example of the pattern that the new rule should warn about?
